### DISCUSSION TITLE
Website for the list

### DISCUSSION THREAD

**Discussion by hng** [hng/tech-coops#145] [General]
I had the plan to celebrate 1k+ stars by setting up a static website for this list to make viewing a bit nicer (and decouple it from Github a bit). The recent HN discussion pushed the number of stars way past 1k, so I got to work again and finished it almost.

You can find a generated version (using mkdocs) on http://tech-coops.xyz now. I would love to get some feedback before I make the website official. I will also add some stuff to the list and do some cleanup. 

Tagging: @decentral1se

**Comment by decentral1se** [comment]
Amazing @hng! I can't see anything on the curent site, just getting a 404?

**Reply by hng** [reply]
Ah thanks! Fixed that issue and it should be back online now :)

**Reply by decentral1se** [reply]
Yes, fantastic!

Some thoughts which you shouldn't feel obliged to do anything about :laughing: 

I'm wondering about the menu split, everything on one page vs. the main left menu. "List of defunct coops" as the only other menu item is a bit confusing but actually all the menu items are in the right menu in the page. Perhaps there is some way to merge the menus? This is purely subjective tho.

![image](https://user-images.githubusercontent.com/1991377/170963498-b0cda87a-ea33-400e-a4dd-a6f9f6711028.png)

Is there a "how to contribute to this list" item somewhere? I can't quite find it on this new listing.

I'm also wondering if "Tech coops list" could be made a little more exciting :upside_down_face: "Workers Co-operatives Directory"? Idk

Thanks for this work, so nice to have this page to point people to!

You might also get some nice feedback on https://community.coops.tech/t/what-hacker-news-thinks-of-tech-co-ops/3201/3 or in a new thread. I think people are interested.

**Reply by hng** [reply]
I've merged the navigation with the table of contents now, I think this way it looks much better and we have more space for the table listing the coops.

There is still the section on contributing at the beginning of the list. In general it is the same README.md shown on GitHub with the manual TOC removed.

As this list does (and should) also include other resources/links besides the list of coops I'd rather keep the name for now.
